index.js

```diff
@@ -90,16 +90,20 @@ Swarmerode._parseSwarmSystemStatus = function (systemStatus) {
   }
 
   for (var i = 0; i < formatted.Nodes; i++) {
-    formatted.ParsedNodes[systemStatus[0][0].trim()] = {
-      Host: systemStatus.shift()[1],
-      Status: systemStatus.shift()[1],
-      Containers: parseInt(systemStatus.shift()[1], 10),
-      ReservedCpus: systemStatus.shift()[1],
-      ReservedMem: systemStatus.shift()[1],
-      Labels: parseLabels(systemStatus.shift()[1]),
-      Error: systemStatus.shift()[1],
-      UpdatedAt: systemStatus.shift()[1],
-      ServerVersion: systemStatus.shift()[1]
+    try {
+      formatted.ParsedNodes[systemStatus[0][0].trim()] = {
+        Host: systemStatus.shift()[1],
+        Status: systemStatus.shift()[1],
+        Containers: parseInt(systemStatus.shift()[1], 10),
+        ReservedCpus: systemStatus.shift()[1],
+        ReservedMem: systemStatus.shift()[1],
+        Labels: parseLabels(systemStatus.shift()[1]),
+        Error: systemStatus.shift()[1],
+        UpdatedAt: systemStatus.shift()[1],
+        ServerVersion: systemStatus.shift()[1]
+      }
+    } catch (err) {
+      debug('ERROR - invalid node', err.message)
```

Comment:
we want all or nothing. outage last night was due to swarm having things in "pending" state which missed a bunch of these keys. since this API is going to change with docker 12, I just put a quick fix for nwo